src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/SendAsyncWinHttpMockHandler.cs

```diff
@@ -4,7 +4,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace System.Net.Http.WinHttpHandlerTests
+namespace System.Net.Http.WinHttpHandlerFunctional.Tests
```

Comment:
I would have used a namespace of `System.Net.Http.WinHttpHandler.Functional.Tests` but it won't compile.  `WinHttpHandler` is a class name and not a namespace.  So, it won't allow using that word as part of a namespace path.